src/components/dataset/introduction.vue

```diff
@@ -32,8 +32,7 @@ except according to the terms contained in the LICENSE file.
           <sentence-separator/>
           <i18n-t keypath="getStarted.full">
             <template #clickHere>
-              <!-- TODO. Specify the `to` prop. -->
-              <doc-link>{{ $t('getStarted.clickHere') }}</doc-link>
+              <doc-link to="central-datasets/">{{ $t('getStarted.clickHere') }}</doc-link>
```

Comment:
The sentence here is specifically:

> For more information on how to get started, **click here**.

I thought about linking specifically to the "Quick Start" section of the page, but I ended up thinking that it made the most sense just to link to the whole page.